src/main/controlMod.F90

```diff
@@ -636,7 +636,7 @@ subroutine control_spmd()
     call mpi_bcast (use_crop, 1, MPI_LOGICAL, 0, mpicom, ier)
     call mpi_bcast (use_fertilizer, 1, MPI_LOGICAL, 0, mpicom, ier)
     call mpi_bcast (use_grainproduct, 1, MPI_LOGICAL, 0, mpicom, ier)
-    call mpi_bcast (use_ozone, 1, MPI_LOGICAL, 0, mpicom, ier)
+    call mpi_bcast (ozone_method, 1, MPI_LOGICAL, 0, mpicom, ier)
```

Comment:
- [ ] Thanks for fixing the type. However, the size (second argument) also needs to be changed: You need to be sure to broadcast the full size of this variable. (As written, I think this will only broadcast the first character.) Look at some of the character variables near the top of this subroutine for examples.

(We should really rewrite this subroutine to use shr_mpi_bcast, which hides these details.)

Reply:
That was embarrassingly sloppy of me. Thank you for pointing this out (again)!